docs/source/guide/tutorials/advanced/semi_sl.rst

```diff
@@ -2,6 +2,141 @@
 Use Semi-Supervised Learning
 ############################
 
-<Guidelines for writing>
+This tutorial provides instructions on how to use semi-supervised learning with OTX.
 
-This tutorial explains how to employ semi-supervised learning with OTX.
+OTX now offers semi-supervised learning, which combines labeled and unlabeled data during training to improve model accuracy. Currently, this type of training is available for multi-class classification, object detection, and semantic segmentation.
+
+Semi-supervised learning will soon be available for multi-label classification and instance segmentation as well.
+
+If want to learn more about the algorithms used in semi-supervised learning, please refer to the explanation section below.
+
+- :doc:`Multi-class Classification <../../explanation/algorithms/classification/multi_class_classification>`
+- :doc:`Object Detection <../../explanation/algorithms/object_detection/object_detection>`
+- :doc:`Semantic Segmentation <../../explanation/algorithms/segmentation/semantic_segmentation>`
+
+In this tutorial, we use MobileNet-V3-large-1x model in multi-class classification to cite an example for semi-supervised learning.
+
+The process has been tested on the following configuration.
+
+- Ubuntu 20.04
+- NVIDIA GeForce RTX 3090
+- Intel(R) Core(TM) i9-10980XE
+- CUDA Toolkit 11.1
+
+
+.. note::
+  
+  Currently, semi-supervised learning models trained in OTX cannot be exported or optimized. However, we will support for this functionality in the near future.
+
+  This tutorial explains how to train a model in semi-supervised learning mode and how to evaluate the resulting model.
+
+
+***************************
+Dataset preparation
+***************************
+
+We use the same dataset, `flowers dataset <https://www.tensorflow.org/hub/tutorials/image_feature_vector#the_flowers_dataset>`_, as we do in :doc:`Classification Tutorial <../base/how_to_train/classification>`.
+
+Since it is assumed that we have a bunch of unlabeled data, which is not annotated but only image files exist in semi-supervised learning,
```

Comment:
```suggestion
Since it is assumed that we have additional unlabeled images,
```
unlabeled == not annotated, we can omit this for simplicity of the sentence